packages/quantic/force-app/main/default/lwc/quanticRecommendationList/templates/grid.html

```diff
@@ -2,16 +2,16 @@
   <template lwc:if={recommendations.length}>
     <div class="slds-text-heading_medium slds-var-p-bottom_x-small">
       <c-quantic-heading
-        data-cy="recommendations__label"
+        data-testid="recommendations__label"
         level={headingLevel}
         label={label}
       ></c-quantic-heading>
     </div>
-    <div class="slds-grid recommendations-list__container">
+    <div class="slds-grid recommendations-list__container" data-testid="recommendations__grid">
```

Comment:
If it's just for unit tests, I would simply add a selector with `.recommendations-list__container` instead of using a `data-testid` here without a corresponding selector in a playwright test.

The reason is to avoid as much as possible to "pollute" our HTML with test selectors unless we need to.